### ISSUE TITLE
Error when calling term.createDocument();

### ISSUE THREAD

**Issue by MJLHThomassen-Sorama** [cronvel/terminal-kit#181]
When i call term.createDocument(); my program immediately throw the following error:

node_modules\terminal-kit\lib\document\Button.js:155
Button.prototype = Object.create( Text.prototype ) ;
                          ^

TypeError: Object prototype may only be an Object or null: undefined.

Im working on windows, but its also throwing this error in my WSL2 (Ubuntu 20.04) terminal.

**Comment by cronvel** [comment]
@MJLHThomassen-Sorama What is your Node.js version?

**Comment by MJLHThomassen-Sorama** [comment]
v14.15.4

**Comment by cronvel** [comment]
@MJLHThomassen-Sorama OK, really strange... 
Can you paste a minimal code that produce this bug?

**Comment by cronvel** [comment]
@MJLHThomassen-Sorama I'm currently running on Node.js v14.15.4 too, and everything works fine on my end...

**Comment by MJLHThomassen-Sorama** [comment]
Yes:

```
const { terminal: term, Palette, Text } = require("terminal-kit");

var document = term.createDocument({
	palette: new Palette()
});

var text = new Text({
	parent: document,
	content: "test",
	x: 10,
	y: 10
});
```

For some reason, when removing Text from the require line and the var text = new Text() line, it does not generate the error (but obviously, also not the text).

**Comment by MJLHThomassen-Sorama** [comment]
Addition:

Strangely enough, if i do it like below, i get no problems, though, the example mentioned in my previous post still gives the same error.

`const termkit = require("terminal-kit");
const term = termkit.terminal;

var document = term.createDocument({
	palette: new termkit.Palette()
});

var text = new termkit.Text({
	parent: document,
	content: "test",
	x: 10,
	y: 10
});
`

**Comment by cronvel** [comment]
@MJLHThomassen-Sorama Thanks for reporting, I will investigate now...

**Comment by cronvel** [comment]
@MJLHThomassen-Sorama Fixed on Terminal-Kit v2.1.6.
Thanks for reporting ;)
